source/_integrations/aten_pe.markdown

```diff
@@ -0,0 +1,59 @@
+---
+title: "ATEN Rack PDU"
+description: "Instructions on how to integrate ATEN Rack PDUs into Home Assistant."
```

Comment:
Missing `logo` frontmatter option.
It makes a huge difference for a user on the documentation if there is a logo. Please consider adding one.

Reply:
There may be copyright or trademark issues involved, which I'd like to avoid.

Reply:
The logo is not used for commercial usage and only services for identification purposes.

Reply:
The logo is not used for commercial usage and only services for identification purposes.

Reply:
I can follow your argument, but I am not a lawyer, and I don't even know which jurisdictions would apply. A person who's comfortable with adding a logo can do it later. I already learned my lessons the hard way.